docs/src/test-snapshots-js.md

```diff
@@ -55,7 +55,8 @@ Sometimes you need to update the reference screenshot, for example when the page
 npx playwright test --update-snapshots
 ```
 
-Note that `snapshotName` is *not a path* relative to the test file, so don't try to use it like `expect(value).toMatchSnapshot('../../test-snapshots/snapshot.png')`.
+> Note that `snapshotName` also accepts a relative path to the snapshot file such as `expect(value).toMatchSnapshot('test-snapshots/snapshot.png')`.
+> However, this path is enfored to be within the snapshots directory for each test file (i.e. `a.spec.js-snapshots`) and will be ignored otherwise.
```

Comment:
- typo: enfored -> enforced
- I'd make it throw when it goes outside of the snapshots directory, and so change this to `However, this path must stay within the snapshots directory for each test file (i.e. a.spec.js-snapshots), otherwise it will throw.`
- A similar comment in `class-testinfo.md` for `outputPath` and `snapshotPath` methods would be great!

Reply:
Good ideas, fixed in e899857.

BTW do I need to manually update `docs/src/test-api/class-testinfo.md` and `docs/src/test-snapshots-js.md` from `types/test.d.ts` or is there some auto generation script I am missing?